spec.emu

```diff
@@ -141,6 +168,24 @@ location: https://github.com/tc39/proposal-measure/
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-amount-reverseroundingmode" type="abstract operation">
+        <h1>ReverseRoundingMode(
+          _mode_: a rounding mode
+        ): a rounding mode
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It computes the rounding mode that would be used if we were to n rounding mode a negative mathematical value.</dd>
```

Comment:
?

Reply:
Ah, that got mangled. I've fixed it now.